CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
     - `generate` to generate the license dart file
 - `checkBeforeGenerate` to run the check command before generate. default = false
 - `downloadPubDevDetails` to get the homepage or repository in the url. default = false
+- `usePubDevZH` to download the pub.dev details from https://pub.flutter-io.cn/ instead of https://pub.dev/. default = false
```

Comment:
Why not allow to specify a url as override

Reply:
fair point